internal_api.go

```diff
@@ -68,6 +96,31 @@ func (e *Enforcer) removePolicy(sec string, ptype string, rule []string) (bool,
 	return ruleRemoved, nil
 }
 
+// removePolicies removes rules from the current policy.
+func (e *Enforcer) removePolicies(sec string, ptype string, rules [][]string) (bool, error) {
+	rulesRemoved := e.model.RemovePolicies(sec, ptype, rules)
```

Comment:
Should call the `RemovePolicy`.

Reply:
@nodece, now you are asking me to change the entire structure. @hsluoyz said to me to follow this structure.

Reply:
It's good.